docs/components/DocumentationSidebarLink.tsx

```diff
@@ -63,9 +64,21 @@ const STYLES_ACTIVE_BULLET = css`
   top: 7px;
 `;
 
-export default class DocumentationSidebarLink extends React.Component {
+type Props = {
+  url?: Url;
+  info: {
+    name: string;
+    as?: string;
+    href: string;
+  };
+  asPath: string;
+};
+
+export default class DocumentationSidebarLink extends React.Component<Props> {
   componentDidMount() {
     // Consistent link behavior across dev server and static export
+    // TODO: Fix this TS somehow
+    // @ts-ignore
```

Comment:
I'm not sure if we even need this anymore with Next 9.5. Would be nice to know what this was used for, and if this is still affecting something with the new Next.